fission-cli/library/Fission/CLI/Handler/App/Publish.hs

```diff
@@ -103,17 +109,40 @@ publish watchFlag runner appURL appPath _updateDNS updateData = do -- TODO updat
               CLI.Error.put err "Server unable to sync data"
               raise err
 
-            Right _ ->
-              case watchFlag of
-                WatchFlag False ->
-                  success appURL
+            Right _ -> do
+              BaseUrl {..} <- getRemoteBaseUrl
+
+              let remoteUrl =
+                    (case baseUrlScheme of
+                        Https -> "https://"
+                        Http  -> "http://"
+                    )
+                    <> "ipfs."
+                    <> baseUrlHost
+                    <> "/ipns/"
+                    -- <> baseUrlPort -- not sure the best way to encorporate the port or if it's needed?
```

Comment:
There's a handy function for this in Servant: [`showBaseUrl`](https://www.stackage.org/haddock/lts-17.10/servant-client-core-0.18.2/Servant-Client-Core.html#v:showBaseUrl).

_However,_ the more I think about this the more I think that we should either use the live app (the `appURL`), or pass a gateway in as a variable in the `config.yaml`.

Reply:
..._or_ have one as a helper in `Remote.hs`

Reply:
> However, the more I think about this the more I think that we should either use the live app (the appURL), or pass a gateway in as a variable in the config.yaml.

Issue is `appURL` doesn't come with a scheme so `openBrowser` will fail trying to open with `file:///User/home/path/to/{appURL}`... 

Personally I find using config.yaml for these one off stringy things really nice. But since the rest of these sorts of things are in that Remote.hs file I'll go with the helper function for now.